### PR TITLE
Bugfix/various improvements

### PR DESCRIPTION
**Pull request by AlexanderAmelkin**
- Fix alignment for log sensor names
- Add CLI mode
- Fix error message for invalid sensor type

### CODE REVIEW

File: list-sensors.cpp [hunk 1]
```diff
@@ -263,7 +263,7 @@ void printSensorData(const std::string& service, const std::string& path)
     size_t folder_pos = path.rfind('/', name_pos - 1);
 
     // row format string
-    constexpr auto row_fmt = "%-18s %8s %7s %-4s %7s %7s %7s %7s %7s\n";
+    constexpr auto row_fmt = "%-19s %8s %7s %-3s %7s %7s %7s %7s %7s\n";
```
Comment (nest1ing) [inline list-sensors.cpp:266]: `Unit` contains 4 characters. As a result, it pushes all other columns headers and looks weird.
```
=== current ===
Name                  Status   Value Unit      LC     LNC     UNC      UC      NR
PSU1_Input_Current        OK   0.367 A       N/A     N/A     N/A     N/A     N/A
PSU1_Output_Current       OK   4.117 A       N/A     N/A 105.000 125.000     N/A
```

Comment (AlexanderAmelkin) [reply list-sensors.cpp:266]: Good catch. I missed that.

Comment (AlexanderAmelkin) [reply list-sensors.cpp:266]: I was wondering why it was 4 and didn't notice that it's the length of the column name

File: list-sensors.cpp [hunk 2]
```diff
@@ -465,8 +504,16 @@ int main(int argc, char* argv[])
     }
     catch (const sdbusplus::exception::SdBusError& ex)
     {
-        fprintf(stderr, "Error: %s\n", ex.what());
-        return EXIT_FAILURE;
+        if (strstr(ex.what(), "org.freedesktop.DBus.Error.FileNotFound"))
```
Comment (nest1ing) [inline list-sensors.cpp:507]: `SdBusError::name()` returns the original exception class.
Likely, the code below would be a little bit better.
```cpp
if (0 == strcmp(ex.name(), "org.freedesktop.DBus.Error.FileNotFound"))
```

**Review by nest1ing — APPROVED** [review]

**Pull request merged**